datafusion/sql/src/select.rs

```diff
@@ -590,44 +589,35 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                 if empty_from {
                     return plan_err!("SELECT * with no tables specified is not valid");
                 }
-                // do not expand from outer schema
-                let expanded_exprs =
-                    expand_wildcard(plan.schema().as_ref(), plan, Some(&options))?;
                 // If there is a REPLACE statement, replace that column with the given
                 // replace expression. Column name remains the same.
                 if let Some(replace) = options.opt_replace {
```

Comment:
I'm considering how to handle the `replace` syntax. I think maybe we can pass `WildcardAdditionalOptions` to `Expr::Wildcard` and also plan it in the analyzer.